tests/test_bucketlist.py

```diff
@@ -0,0 +1,120 @@
+import json
+
+import unittest 
+from tests.test_base import BaseTestCase
+
+class TestAuth(BaseTestCase):
+	# Ensure that cannot create a bucketlist without authorization
+	def test_cannot_post_bucketlist_without_token(self):
+		self.bucketlist = {"title" : 'Going to the moon',
+			"description" : 'I wish to have gone to the moon by 2015'
+			
+		}
+		response = self.client.post('/bucketlists/',
+			data=json.dumps(self.bucketlist),
+			headers={'Content-Type':'application/json'} 
+		)
+
+		output = json.loads(response.data)
+		self.assertTrue("Error: You are not authorized to access this resource." in output['message'])
+		self.assertTrue(response.status_code, 403)
```

Comment:
The more appropriate status code to return here would be a 401 since no token means you're not authorized.